server/src/index.ts

```diff
@@ -1,2 +1,119 @@
-const message = 'Hello World!';
-console.log(message);
+import { Challenge, Status } from './types/interfaces';
```

Comment:
Just an observation: you have too much of a logic in an `index.js` file. It should not be so